src/layout/Modal.js

```diff
@@ -20,18 +20,20 @@ let Modal = ({ isOpen, children, style = {} }) => (
           ...style,
         }}
         onClick={F.off(isOpen)}
+        className="default-modal-bg"
       >
         <div
           style={{
             backgroundColor: '#fff',
             borderRadius: 5,
             maxWidth: 'fit-content',
+            padding: 30,
```

Comment:
let's just kill the default padding, border radius, color, etc that really belongs in grey vest (and so we don't need !important)